src/components/Button/index.stories.js

```diff
@@ -149,7 +150,7 @@ FullWidth.args = {
 export const Icons = () => (
   <StoryWrapper>
     <StoryItem>
-      <Button IconLeft={StarFilledIcon} label="Button" />
+      <Button IconLeft={WorldIcon} label="Button" />
```

Comment:
`WorldIcon` was one of the svg files that contained a forbidden attribute; using it here made it easier to check if our svgo config was working as expected.